samples/MQTTSN/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.10)
+
+PROJECT(azure-iot-udp-samples)
+
+add_subdirectory(${PROJECT_SOURCE_DIR}/lib/azure-sdk-for-c)
+add_subdirectory(${PROJECT_SOURCE_DIR}/lib/paho.mqtt-sn.embedded-c/MQTTSNPacket/src)
+add_subdirectory(${PROJECT_SOURCE_DIR}/lib/paho.mqtt-sn.embedded-c/MQTTSNPacket/samples)
+
+add_executable(sample_qos_0 ${PROJECT_SOURCE_DIR}/src/paho_iot_hub_telemetry_example_qos_0.c)

```

Comment:
How about having one example and use macro to decide QoS0/Qos1

Reply:
Should we also provide the option of passing it through the program arguments? like ./sample_telemetry [qos] [host address] [host port]

Reply:
Lets have macro that when defined allow user to send QoS0 and if not define default back to QoS1. 

---
In reply to: [450147728](https://github.com/t-meitan-msft/azure-iot-udp-samples/pull/1#discussion_r450147728) [](ancestors = 450147728)